### PR TITLE
List all sub-statuses

### PR DESCRIPTION
**Pull request by adamlazik1**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (planned Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (Satellite 6.13)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* We do not accept PRs for Foreman older than 3.1.

### CODE REVIEW

**Comment by adamlazik1** [comment]
This is still heavily work-in-progress, but regardless any feedback is welcome.

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 1]
```diff
@@ -3,12 +3,27 @@
...
+* Service level
+* Role
+* Usage
+* Addons
+* System purpose
+* Traces
+endif::[]
 
-The *configuration* sub-status is only relevant if {Project} uses a configuration management system like Ansible, Puppet, or Salt.
+There can be more sub-statuses depending on which plugins you add to your {Project}.
```
Comment (maximiliankolb) [inline guides/common/modules/con_host-substatus-overview.adoc:26]:
```suggestion
There can be more sub-statuses depending on which plug-ins you add to your {Project}.
```

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 2]
```diff
@@ -3,12 +3,27 @@
 
 A sub-status monitors only part of a host's capabilities.
 
-Currently, {Project} ships only with *Build* and *Configuration* sub-statuses.
-There can be more sub-statuses depending on which plugins you add to your {Project}.
-
-The *build* sub-status is relevant for managed hosts and when {Project} runs in unattended mode.
+Currently, {Project} ships with the following sub-statuses:
```
Comment (maximiliankolb) [inline guides/common/modules/con_host-substatus-overview.adoc:6]:
```suggestion
{Project} contains the following sub-statuses:
```

**Comment by adamlazik1** [comment]
@ekohl would you happen to know if I correctly distinguished between substatuses for all builds and for builds with katello?

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 3]
```diff
@@ -1,55 +1,247 @@
...
 
-Currently, {Project} ships only with *Build* and *Configuration* sub-statuses.
-There can be more sub-statuses depending on which plugins you add to your {Project}.
+{Project} contains the following sub-statuses:
 
-The *build* sub-status is relevant for managed hosts.
+Configuration::
+This sub-status is only relevant if {Project} uses a configuration management system like Ansible, Puppet, or Salt.
++
+Possible values:
```
Comment (ekohl) [inline guides/common/modules/con_host-substatus-overview.adoc:11]: I think the values defined are here:
https://github.com/theforeman/foreman/blob/b5e5dfb4108d36210a1691af45d6d8b980f4f6d1/app/models/host_status/configuration_status.rb#L4
So https://github.com/theforeman/foreman/blob/b5e5dfb4108d36210a1691af45d6d8b980f4f6d1/app/models/config_report.rb#L2

Perhaps the labels are a better way:
https://github.com/theforeman/foreman/blob/b5e5dfb4108d36210a1691af45d6d8b980f4f6d1/app/models/host_status/configuration_status.rb#L65-L83

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 4]
```diff
@@ -1,55 +1,247 @@
...
++
+[options="header"]
+|===
+| Value | Global Host Status | Number
+| Active | OK | 0
+| Pending | OK | 1
+| No changes | OK | 2
+| No reports | OK / Warning | 3
+| Out of sync | Warning | 4
+| Error | Error | 5
```
Comment (ekohl) [inline guides/common/modules/con_host-substatus-overview.adoc:22]: I don't think there are numbers. At most there are bit numbers (since the total value is a number and things are calculated bitwise.

You probably convert https://github.com/theforeman/foreman/blob/b5e5dfb4108d36210a1691af45d6d8b980f4f6d1/app/models/host_status/configuration_status.rb#L43-L59

reading this, wouldn't it make more sense to flip it and start with the global status and explain what may be the case?

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 5]
```diff
@@ -1,55 +1,247 @@
...
+|===
++
+Additional information about the sub-status values:
```
Comment (ekohl) [inline guides/common/modules/con_host-substatus-overview.adoc:25]: Reading the source I get the impression that `ConfigurationStatus` doesn't have a substatus.

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 6]
```diff
@@ -1,55 +1,247 @@
...
+When there are no reports but the host uses an associated configuration management proxy or the `always_show_configuration_status` setting is set to `true`, it maps to *Warning*.
+Otherwise it maps to *OK*.
+* *Error*: This indicates an error during configuration.
+For example, a run failed to install a package.
+* *Out of sync*: A configuration report was not received within the expected interval, based on the `outofsync_interval` setting.
+Reports are identified by an origin and can have different intervals based upon it.
 
-The *configuration* sub-status is only relevant if {Project} uses a configuration management system like Ansible, Puppet, or Salt.
+Build::
+This sub-status is only relevant for managed hosts, hosts registered through global registration, and when {Project} runs in unattended mode.
```
Comment (ekohl) [inline guides/common/modules/con_host-substatus-overview.adoc:39]: Unattended mode is gone and now always one.
```suggestion
This sub-status is only relevant for managed hosts or hosts registered through global registration.
```

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 7]
```diff
@@ -1,55 +1,247 @@
...
++
+Possible values:
+// TODO: numbers
```
Comment (ekohl) [inline guides/common/modules/con_host-substatus-overview.adoc:42]: Here it's easier to understand what's going on:
https://github.com/theforeman/foreman/blob/b5e5dfb4108d36210a1691af45d6d8b980f4f6d1/app/models/host_status/build_status.rb#L3-L10

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 8]
```diff
@@ -1,55 +1,247 @@
...
+| Built | OK | 0
+|===
 
-To view the sub-status for a host, in the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and click the host whose full status you want to inspect.
-You can also view substatus information in the hover help for each host.
 
-In the *Properties* table of the host details' page, you can view both the global host status and all sub-statuses.
+ifdef::satellite,orcharhino[]
+Execution::
+Status of the last completed remote excution job or ansible job.
```
Comment (ekohl) [inline guides/common/modules/con_host-substatus-overview.adoc:54]: AFAIK Ansible jobs are REX jobs. It's also available everywhere, but with `foreman-el` and `foreman-deb` you can add a note that you need to have the REX plugin installed. With others it's always present.

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 9]
```diff
@@ -1,55 +1,247 @@
...
++
+Possible values:
+// TODO: Values, numbers
```
Comment (ekohl) [inline guides/common/modules/con_host-substatus-overview.adoc:57]: https://github.com/theforeman/foreman_remote_execution/blob/master/app/models/host_status/execution_status.rb

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 10]
```diff
@@ -1,55 +1,247 @@
...
++
+[options="header"]
+|===
+| Value | Global Host Status | Number
+| Job completed succssfully | OK | 0
+| Job not completed successfully | Error | 0
+|===
 
-Each sub-status can define its own set of possible values that are mapped to the three global status values.
+Compliance::
```
Comment (ekohl) [inline guides/common/modules/con_host-substatus-overview.adoc:66]: Compliance should be hidden on `foreman-deb`. On others it's always optional AFAIK.

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 11]
```diff
@@ -1,55 +1,247 @@
...
+Indicates if the host is compliant with OpenSCAP policies.
++
+Possible values:
```
Comment (ekohl) [inline guides/common/modules/con_host-substatus-overview.adoc:69]: https://github.com/theforeman/foreman_openscap/blob/master/app/models/foreman_openscap/oval_status.rb

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 12]
```diff
@@ -1,55 +1,247 @@
...
+[options="header"]
+|===
+| Value | Global Host Status | Number
+| Compliant | OK | 0
+| Inconclusive | Warning | 1
+| At least one incompliant | Error | 2
+|===
 
-The *Build* sub-status has two possible values {endash} *pending* and *built* that are both mapped to global *OK* value.
+Inventory::
```
Comment (ekohl) [inline guides/common/modules/con_host-substatus-overview.adoc:79]: The rh_cloud plugin depends on Katello. I don't know if orcharhino ships it. Satellite enables it by default, in Katello it's optional.

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 13]
```diff
@@ -1,55 +1,247 @@
...
+Indicates if the host is synchronized to {RHCloud}.
+{ProjectServer} performs the this synchronization itself but only uploads basic information to {RHCloud}.
++
+Possible values:
```
Comment (ekohl) [inline guides/common/modules/con_host-substatus-overview.adoc:83]: https://github.com/theforeman/foreman_rh_cloud/blob/foreman_3_9/app/models/inventory_sync/inventory_status.rb

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 14]
```diff
@@ -1,55 +1,247 @@
...
++
+[options="header"]
+|===
+| Value | Global Host Status | Number
+| Disconnect | Warning | 0
+| Sync | OK | 1
+|===
 
-The *Configuration* status has more possible values that map to the global status as follows:
+Insights::
```
Comment (ekohl) [inline guides/common/modules/con_host-substatus-overview.adoc:92]: This is also provided by the rh_cloud, so the same conditions as above apply.

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 15]
```diff
@@ -1,55 +1,247 @@
...
+Indicates if the host is synchronized to {RHCloud}.
+This synchronization is performend by the host.
+The host uploads more information than the {ProjectServer}.
++
+Possible values:
```
Comment (ekohl) [inline guides/common/modules/con_host-substatus-overview.adoc:97]: https://github.com/theforeman/foreman_rh_cloud/blob/foreman_3_9/app/models/insights_client_report_status.rb

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 16]
```diff
@@ -1,55 +1,247 @@
...
++
+[options="header"]
+|===
+| Value | Global Host Status | Number
+| Reporting | OK | 0
+| No report | Error | 1
+|===
 
-.Sub-statuses that map to the global *OK* status
+RHEL Lifecycle::
```
Comment (ekohl) [inline guides/common/modules/con_host-substatus-overview.adoc:106]: This comes from Katello

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 17]
```diff
@@ -1,55 +1,247 @@
...
+Indicates the current state of the {RHEL} operating system installed of the host.
++
+Possible values:
```
Comment (ekohl) [inline guides/common/modules/con_host-substatus-overview.adoc:109]: https://github.com/Katello/katello/blob/master/app/models/katello/rhel_lifecycle_status.rb

**Review by ekohl — COMMENTED** [review]
I'd be very hesitant to publish the numbers. I'd consider those internal values and inside the application we shouldn't use them. If we do, that's IMHO something we should change.

As for the unannotated parts, AFAIK they all come from Katello.

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 18]
```diff
@@ -1,55 +1,247 @@
 [id="host-substatus-overview_{context}"]
 = Host Sub-status Overview
 
-A sub-status monitors only part of a host's capabilities.
+A sub-status monitors only a part of host's capabilities.
```
Comment (bangelic) [inline guides/common/modules/con_host-substatus-overview.adoc:4]:
```suggestion
A sub-status monitors only a part of a host's capabilities.
```

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 19]
```diff
@@ -1,55 +1,247 @@
...
+| Value | Global Host Status | Number
+| Compliant | OK | 0
+| Inconclusive | Warning | 1
+| At least one incompliant | Error | 2
+|===
 
-The *Build* sub-status has two possible values {endash} *pending* and *built* that are both mapped to global *OK* value.
+Inventory::
+Indicates if the host is synchronized to {RHCloud}.
+{ProjectServer} performs the this synchronization itself but only uploads basic information to {RHCloud}.
```
Comment (bangelic) [inline guides/common/modules/con_host-substatus-overview.adoc:81]:
```suggestion
{ProjectServer} performs the synchronization itself but only uploads basic information to {RHCloud}.
```

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 20]
```diff
@@ -1,55 +1,247 @@
...
+[options="header"]
+|===
+| Value | Global Host Status | Number
+| Reporting | OK | 0
+| No report | Error | 1
+|===
 
-.Sub-statuses that map to the global *OK* status
+RHEL Lifecycle::
+Indicates the current state of the {RHEL} operating system installed of the host.
```
Comment (bangelic) [inline guides/common/modules/con_host-substatus-overview.adoc:107]:
```suggestion
Indicates the current state of the {RHEL} operating system installed on the host.
```

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 21]
```diff
@@ -1,55 +1,247 @@
...
+| Approaching end of support | Warning | 5
+| Support ended | Error | 6
+|===
 
-Active::
-During the last run, some resources were applied.
+endif::[]
+ifdef::katello,satellite,orcharhino[]
+Errata::
+Indicates if Errata available on the host.
```
Comment (bangelic) [inline guides/common/modules/con_host-substatus-overview.adoc:126]:
```suggestion
Indicates if Errata is available on the host.
```

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 22]
```diff
@@ -1,55 +1,247 @@
...
+|===
+| Value | Global Host Status | Number
+| Unknown | OK | 0
+| Mismatched | Warning | 1
+| Matched | OK | 2
+| Not specified | OK | 3
+|===
+
+Traces::
+Indicates if the host need a reboot or a process restart.
```
Comment (bangelic) [inline guides/common/modules/con_host-substatus-overview.adoc:221]:
```suggestion
Indicates if the host needs a reboot or a process restart.
```

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 23]
```diff
@@ -1,55 +1,247 @@
...
+| Required reboot | Error | 2
+|===
+endif::[]
+
+There can be more sub-statuses depending on which plug-ins you add to your {Project}.
+
+To view the sub-status for a host, in the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and click the host whose full status you want to inspect.
+You can also view substatus information in the hover help for each host.
+
+In the *Properties* table of the host details' page, you can view both the global host status and all sub-statuses.
```
Comment (bangelic) [inline guides/common/modules/con_host-substatus-overview.adoc:240]:
```suggestion
In the *Properties* table of the host details page, you can view both the global host status and all sub-statuses.
```

**Review by bangelic — CHANGES_REQUESTED** [review]

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 5 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc:25]: I meant that Configuration status itself is a host sub-status, will try to reword to convey the message better.

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 9 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc]: Thank you for linking the file, but I am a bit perplexed here. both QUEUED and RUNNING have label "Unknown execution status", is this what is actually displayed in the UI? Also does that mean that this value can have both 2 and 3 assigned as number depending on what the actual status is?

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 10 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc:75]: Fixed, thank you.

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 11 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc:78]: So OVAL status and Compliance status are the same?

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 16 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc:117]: Thank you, fixed.

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 15 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc:108]: Changed "No report" to "Not reporting" based on the linked file.

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 17 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc:120]: Thank you for linking the file.
What what I am seeing, everything in this status should be mapped correctly.

**Review by adamlazik1 — COMMENTED** [review]
@ekohl @bangelic thank you both for your review. While implementing the feedback some new questions rose which I left as responses to review comments. @ekohl do you think you could help me answer these?

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 18 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc]: Applied.

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 19 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc]: Applied.

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 20 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc]: Applied.

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 21 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc]: Applied.

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 22 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc]: Applied.

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 23 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc]: Applied.

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 13 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc:94]: Fixed values.

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 12 above)
Comment (maximiliankolb) [reply guides/common/modules/con_host-substatus-overview.adoc:90]: Yes, orcharhino ships RH cloud too. It's optional on orcharhino.

Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc:90]: Thanks for confirming.

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 4 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc:21]: Edit: I misunderstood the comment. Removing numbers from this sub-status.

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 24]
```diff
@@ -1,55 +1,267 @@
...
+| Pending | OK
+| No changes | OK
+| No reports | OK / Warning
+| Out of sync | Warning
+| Error | Error
+|===
++
+Additional information about the values of this sub-status:
++
+* *Active*: During the last run, some resources were applied.
```
[the remark this answers is not in the document]
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc]: Applied.

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 25]
```diff
@@ -1,55 +1,267 @@
...
+When there are no reports but the host uses an associated configuration management proxy or the `always_show_configuration_status` setting is set to `true`, it maps to *Warning*.
+Otherwise it maps to *OK*.
+* *Error*: This indicates an error during configuration.
+For example, a run failed to install a package.
+* *Out of sync*: A configuration report was not received within the expected interval, based on the `outofsync_interval` setting.
+Reports are identified by an origin and can have different intervals based upon it.
 
-In the *Properties* table of the host details' page, you can view both the global host status and all sub-statuses.
+Build::
+This sub-status is only relevant for managed hosts or hosts registered through global registration.
```
[the remark this answers is not in the document]
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc]: Changed to hosts provisioned by {Project}.

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 26]
```diff
@@ -1,55 +1,267 @@
...
+| Error | Error
+|===
++
+Additional information about the values of this sub-status:
++
+* *Active*: During the last configuration, some resources were applied.
+* *Pending*: During the last configuration, some resources would be applied but your configuration management integration was configured to run in `noop` mode.
+* *No changes*: During the last configuration, nothing changed.
+* *No reports*: This can be both a *Warning* or *OK* status.
+When there are no reports but the host uses an associated configuration management proxy or the `always_show_configuration_status` setting is set to `true`, it maps to *Warning*.
```
Comment (maximiliankolb) [inline guides/common/modules/con_host-substatus-overview.adoc:39]: what do you mean by "proxy"? Maybe reword to "{SmartProxy} for configuration management"?

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 27]
```diff
@@ -1,55 +1,267 @@
...
+| Up to date | OK | 0
+| Unknown | Warning | 1
+| Needed errata | Error | 2
+| Needed security errata | Error | 3
+|===
 
-No reports::
-This can be both a *Warning* or *OK* sub-status.
-This occurs when there are no reports but the host uses, for example, an associated configuration management proxy or `always_show_configuration_status` setting is set to `true`, it maps to *Warning*.
+Subscription::
```
Comment (maximiliankolb) [inline guides/common/modules/con_host-substatus-overview.adoc:165]: RHEL subscription or Foreman/candlepin subscription?

**Review by maximiliankolb — COMMENTED** [review]
two small comments, rest LGTM

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 27 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc:165]: Red-hat subscription acquired through subscription-manager. Should I make it satellite-only?

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 26 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc]: Reworded.

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 27 above)
Comment (maximiliankolb) [reply guides/common/modules/con_host-substatus-overview.adoc:165]: No, then it'll also affect Foreman/orcharhino. Instead, I would reword this to "RHEL subscription" if possible.

**Comment by adamlazik1** [comment]
I tested on the Foreman 3.9 and turns some some other substatuses are available on foreman as well. I adjusted the ifdefs and ordered substatuses as they are ordered in the web ui.

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 27 above)
Comment (maximiliankolb) [reply guides/common/modules/con_host-substatus-overview.adoc:152]: I would assume "no" because you don't need a valid subscription for Debian. But you can manage RHEL hosts using Foreman/orcharhino.

I have checked on Foreman 3.9 with SCA off for AlmaLinux 8:

Screenshot from 2024-02-21 16-04-23
 
with SCA on, it's green by default (Debian 12):

image

**Comment by adamlazik1** [comment]
@maximiliankolb could you verify if OVAL scan is available on orcharhino?

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 27 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc:152]: I guess I should keep it visible for foreman-deb if it's present.

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 28]
```diff
@@ -1,55 +1,271 @@
...
-A sub-status monitors only part of a host's capabilities.
+A sub-status monitors only a part of a host's capabilities.
 
-Currently, {Project} ships only with *Build* and *Configuration* sub-statuses.
-There can be more sub-statuses depending on which plug-ins you add to your {Project}.
+To view the sub-statuses of a host, in the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and click on the host whose full status you want to inspect.
+You can view the global host status next to the name of the host and the host sub-statuses on the *Host status* card.
 
-The *build* sub-status is relevant for managed hosts.
+Each sub-status can define its own set of possible values that are mapped to the three global status values.
```
Comment (maximiliankolb) [inline guides/common/modules/con_host-substatus-overview.adoc:9]:
```suggestion
Each sub-status defines its own set of possible values that are mapped to the three global status values.
```

alternative: "has"?

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 29]
```diff
@@ -1,55 +1,271 @@
...
+|===
+| Label | Global host status | Number value
+| Compliant | OK | 0
+| Inconclusive | Warning | 1
+| At least one incompliant | Error | 2
+|===
+endif::[]
 
-The *Build* sub-status has two possible values {endash} *pending* and *built* that are both mapped to global *OK* value.
+ifdef::satellite,orcharhino[]
```
Comment (maximiliankolb) [inline guides/common/modules/con_host-substatus-overview.adoc:75]:
```suggestion
ifndef::foreman-deb[]
```

Unless OVAL is not connected to "foreman_openscap"? https://docs.theforeman.org/nightly/Managing_Security_Compliance/index-katello.html#Security_Content_Automation_Protocol_security-compliance

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 30]
```diff
@@ -1,55 +1,271 @@
...
+| Unknown execution status | OK | 2 or 3
+| Last execution cancelled | OK | 4
+|===
 
-.Sub-statuses that map to the global *OK* status
+ifdef::satellite,orcharhino,katello[]
+Inventory::
+Indicates if the host is synchronized to {RHCloud}.
+{ProjectServer} performs the synchronization itself but only uploads basic information to {RHCloud}.
++
```
Comment (maximiliankolb) [inline guides/common/modules/con_host-substatus-overview.adoc:112]:
```suggestion
+
ifdef::orcharhino[]
+
Only applies if you have the Red Hat Cloud plug-in installed.
endif::[]
```

See https://docs.theforeman.org/3.9/Managing_Hosts/index-orcharhino.html#Installing_Red_Hat_Cloud_Plug_in_managing-hosts

You can also drop "katello" from line 108.

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 31]
```diff
@@ -1,55 +1,271 @@
...
+[options="header"]
+|===
+| Label | Global host status | Number value
+| Reporting | OK | 0
+| Not reporting | Error | 1
+|===
 
-Pending::
-During the last run, some resources would be applied but your configuration management integration was configured to run in `noop` mode.
+Errata::
```
Comment (maximiliankolb) [inline guides/common/modules/con_host-substatus-overview.adoc:136]: Errata should be "katello,satellite,orcharhino". please add "endif" here and another "ifdef:katello,satellite,orcharhino" below. see my other comment above.

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 27 above)
Comment (maximiliankolb) [reply guides/common/modules/con_host-substatus-overview.adoc:152]: If should only be visible for katello,satellite,orcharhino because you can only do content management with Katello plug-in. However, the "Subscription" status refers to the client OS and appears to be valid for all registered hosts (not just RHEL) AFAIK.

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 29 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc:75]: Well, it is not present on katello build, so I presume it's not gonna be on bare foreman.

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 27 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc:154]: So leaving visibility only for katello,satellite,orcharhino.

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 28 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc]: Applied.

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 30 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc:116]: Applied.

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 29 above)
Comment (adamruzicka) [reply guides/common/modules/con_host-substatus-overview.adoc:75]: oval comes from foreman_openscap

Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc:75]: @maximiliankolb I'd rather omit it since it is not on bare foreman unless you specifically want me to make it visible.

**Comment by adamlazik1** [comment]
Any other feedback or is this good to merge?

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 32]
```diff
@@ -1,55 +1,275 @@
...
+| Host was not uploaded to your RH cloud inventory | Warning | 0
+| Successfully uploaded to your RH cloud inventory | OK | 1
+|===
 
-Active::
-During the last run, some resources were applied.
+Insights::
+Indicates if the host is synchronized to {RHCloud}.
+This synchronization is performend by the host.
+The host uploads more information than the {ProjectServer}.
```
Comment (maximiliankolb) [inline guides/common/modules/con_host-substatus-overview.adoc:129]: Do you mean "_to_ your {ProjectServer}"?

**Review by maximiliankolb — COMMENTED** [review]
One tiny question.

File: guides/common/modules/con_host-substatus-overview.adoc (hunk 32 above)
Comment (adamlazik1) [reply guides/common/modules/con_host-substatus-overview.adoc:129]: No. Both host and the project server upload information to RH cloud, but project server only uploads basic info, whereas host uploads more extensive info.

**Comment by maximiliankolb** [comment]
Then it's good to go from my side. Don't forget to cherry-pick to "3.9" and "3.10". Seems like you've opened this PR just before branching 3.9 :joy:

**Comment by adamlazik1** [comment]
Merged and cherry-picked:
   f327fb27ba..80c41c05ba  3.10 -> 3.10
   96455ab526..ed21f5e5ef  3.9 -> 3.9

**Comment by adamlazik1** [comment]
For 3.8 and below: (removed RHEL lifecycle)
https://github.com/theforeman/foreman-documentation/pull/2835
